astropy/io/ascii/tests/test_read.py

```diff
@@ -231,31 +233,31 @@ def test_daophot_multiple_aperture2():
 @pytest.mark.parametrize('fast_reader', [True, False, 'force'])
 def test_empty_table_no_header(fast_reader):
     with pytest.raises(ascii.InconsistentTableError):
-        table = ascii.read('t/no_data_without_header.dat', Reader=ascii.NoHeader,
-                            guess=False, fast_reader=fast_reader)
+        ascii.read('t/no_data_without_header.dat', Reader=ascii.NoHeader,
+                   guess=False, fast_reader=fast_reader)
 
 
 @pytest.mark.parametrize('fast_reader', [True, False, 'force'])
 def test_wrong_quote(fast_reader):
-    with pytest.raises(ascii.InconsistentTableError):
-        table = ascii.read('t/simple.txt', guess=False, fast_reader=fast_reader)
+    with pytest.raises((ascii.InconsistentTableError, cparser.CParserError)):
```

Comment:
Why do we need to catch that other error? That's an API change, because users might have code that tests for `InconsistentTableError`. For example, I do something like:
```
try:
    tab = Table.read(... parameters that are appropriate for a table with format version 1 ...)
except InsonsistentTableError:
    tab = Table.read(... parameters for table with format version 2 ... )
```
That won't work, if we suddenly throw a different exception. (I my particular case, I don't have to worry too much, I'm just saying that I might not be the only one doing that.)

Reply:
Fixed.